src/js/components/Card/stories/RichFooter.js

```diff
@@ -99,5 +98,3 @@ const Example = () => {
     </Grommet>
   );
 };
```

Comment:
In order to not avoid changes to the name of the story you can use the story name parameter
```suggestion
};

RichFooter.story = { name: 'Rich footer' };
```